docs/Developers/Review.rst

```diff
@@ -132,6 +132,8 @@ For the Reviewers
    Bears require special attention during testing.
    `Testing Bears <http://api.coala.io/en/latest/Developers/Testing_Bears.html>`__
    provides a guideline for how to test bears.
+-  Check that the changes haven't been pushed from master branch.
+   Although it's not mandatory but a good practice to follow.
```

Comment:
This sentence needs to be improved so it is easier to understand.

Reply:
You might change it to `As a good practice, check if the changes in PR are not pushed from the master branch`

Reply:
@jayvdb  Sentence updated as per mentioned by @yashovardhanagrawal. Please merge the PR